nixos/tests/ceph-multi-node.nix

```diff
@@ -0,0 +1,231 @@
+import ./make-test.nix ({pkgs, lib, ...}:
+
+let
+  generateCephConfig = { daemonConfig }: {
+    enable = true;
+    global = {
+      fsid = "066ae264-2a5d-4729-8001-6ad265f50b03";
+      monInitialMembers = "a";
+      monHost = "192.168.1.1";
+    };
+  } // daemonConfig;
+
+  generateHost = { pkgs, cephConfig, networkConfig, ... }: {
+    virtualisation = {
+      memorySize = 1536;
+      emptyDiskImages = [ 20480 ];
```

Comment:
Does this actually get allocated up front? That might be a little scary.

Reply:
I think its just a "up to" specification. The machines are allowed to use up to 1.5GB RAM, but that could probably be tweaked.
I doubt the machines actually need more than 500-700 MB of RAM.

Reply:
Yeah, I just tried it, and in totalt the entire test used ~1.5GB RAM at the most. So I guess we could tweak this to a lower allocation. 512MB per machine should be sufficient.